db/migrate/20161101133602_add_currency_column_to_payment_settings.rb

```diff
@@ -0,0 +1,16 @@
+class AddCurrencyColumnToPaymentSettings < ActiveRecord::Migration
+  def up
+    add_column :payment_settings, :minimum_price_currency, :text, after: :minimum_price_cents
```

Comment:
The column type is now `text`, which is way too much for a single currency (65535 characters). You could use varchar here.